infra/main.bicep

```diff
@@ -134,7 +91,6 @@ module quoteOfTheDay './app/QuoteOfTheDay.bicep' = {
   scope: rg
 }
 
-output AZURE_SPLIT_WORKSPACE_NAME string = splitExperimentationWorkspace.outputs.splitExperimentationWorkspaceName
 output AZURE_APPCONFIGURATION_NAME string = appConfiguration.outputs.appConfigurationName
```

Comment:
I think you can remove the `AZURE_APPCONFIGURATION_NAME` line, it was only used in the `azure.yaml` code we're removing